config/routes.rb

```diff
@@ -1,22 +1,20 @@
 Rails.application.routes.draw do
+  root to: "admins#show"
+  resource :admin, only: [:show]
+
   namespace :admin do
     resources :categories, only: [:index, :create, :destroy]
     resources :orders, only: [:index, :show, :update]
-    resources :items, only: [:index, :new, :create, :edit, :update]
+    resources :items, except: [:destroy]
   end
 
-  resource :admin, only: [:show]
-
-  resources :items, only: [:index, :show]
-  resources :orders, only: [:create, :show, :index]
-  resource :cart, only: [:create, :show, :update, :destroy]
-
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
+  delete '/logout', to: 'sessions#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.